.golangci.yml

```diff
@@ -143,6 +143,7 @@ linters:
     - interfacer
     - ireturn
     - lll # Think about adding this (max line length)
+    - maintidx
```

Comment:
new linter, similar to "complexity" linters that we disable.

Reply:
so the default is `20`, so ideally any function with maintainability index > 20 should raise linter errors